spec/features/user_posts_photos_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.feature 'Users can post photos to the timeline', type: :feature do
+  scenario 'Uploaded images appear on the next screen' do
+    sign_in
+    click_link 'New post'
+    attach_file('post_image', Rails.root + 'public/owl.jpg')
+    fill_in 'Message', with: 'Test post'
+    click_button 'Submit'
+    expect(page).to have_xpath("//img[contains(@src,'owl.jpg')]")
```

Comment:
CSS selectors might be more common than xpath selectors here —

```ruby
expect(page).to have_css("img[src*='owl.jpg']")
```

In this case I think the XPath is more readable, so I'd leave it in, but be aware CSS selectors are usually preferred.